src/server/models/news/router.js

```diff
@@ -0,0 +1,47 @@
+const router = require('express').Router();
+
+const path = require('path');
+const fs = require('fs');
+const util = require('util');
+const News = require('./model');
+
+const readFileAsync = util.promisify(fs.readFile);
+const writeFileAsync = util.promisify(fs.writeFile);
+
+const storageFilePath = path.resolve(__dirname, '../data.json');
+
+router.route('/api/my-news/:id?')
+    .get((req, res, next) => {
+        if (req.params.id !== undefined) {
+            News.findById(req.params.id)
```

Comment:
Can you use facade to isolate schemas in controllers??

Reply:
Straight using schemas in controllers aren't good

Reply:
fixed